otcdocstheme/theme/otcdocs/header.html

```diff
@@ -19,7 +19,7 @@
         <img src="{{pathto('_static/images/deutsche-telekom-logo.svg', 1)}}" />
       </a>
       {% if theme_site_name %}
-      <div class="docs-title justify-content-center">{{ theme_site_name }}</div>
+      <a class="docs-title justify-content-center" href="{{ base_url }}" aria-label="{{ theme_site_name }}">{{ theme_site_name }}</a>
```

Comment:
and why do you want this?

Reply:
I am used to click on the main headline to get back to the home page of the specific website. I am never clicking on the logo to get back. For having both options I added the **href part to our title**.

Reply:
but the same is already on clicking the doc title in the left menu. Too many links imho

Reply:
No, it's not. The click on the title of the left menu leads to the page where the specific document is located like ECS UMN. The title link leads to the docsportal page, the main-page over all documents which I would also expect under clicking on the main-header.